### PR TITLE
Uplink CLI setup welcome message

### PR DESCRIPTION
**Pull request by mniewrzal**
What: Additional welcome message when `uplink setup` wizard is finished
image

Why: Better user experience during setup 

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: cmd/uplink/cmd/setup.go [hunk 1]
```diff
@@ -115,9 +115,27 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 			"api-key":        apiKey,
 			"enc.key":        string(encKey),
 		}
+
+		fmt.Println()
+		fmt.Println("Your Uplink CLI is configured and ready to use!")
```
Comment (jtolds) [inline cmd/uplink/cmd/setup.go]: this should probably be below the actual process.SaveConfig call

Comment (mniewrzal) [reply cmd/uplink/cmd/setup.go]: done

File: cmd/uplink/cmd/setup.go [hunk 2]
```diff
@@ -71,19 +71,37 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
...
 		if err != nil {
 			return err
 		}
+		satellites := []string{"mars.tardigrade.io", "jupiter.tardigrade.io", "saturn.tardigrade.io"}
 		var satelliteAddress string
 		_, err = fmt.Scanln(&satelliteAddress)
 		if err != nil {
-			return err
+			// fmt.Scanln cannot handle empty input
+			satelliteAddress = satellites[0]
```
Comment (ifraixedes) [inline cmd/uplink/cmd/setup.go]: we can detect an empty input just checking `err != nil && n == 0` where `n` is from `n, err = fmt.Scanln(...`

We could use that to show an error saying that `satelliteAddress` is required and stop or loop over until one option is selected or a manual address is written.

Comment (mniewrzal) [reply cmd/uplink/cmd/setup.go]: added conditions about `n == 0`

We can add loop later if needed.

File: cmd/uplink/cmd/setup.go [hunk 3]
```diff
@@ -114,17 +118,17 @@ Please enter numeric choice or enter satellite address manually [1]: `)
...
 		}
 
 		if apiKey == "" {
 			return errs.New("API key cannot be empty")
 		}
 
-		_, err = fmt.Print("Enter your encryption passphrase: ")
-		if err != nil {
+		n, err = fmt.Print("Enter your encryption passphrase: ")
+		if err != nil && n != 0 {
```
Comment (ifraixedes) [inline cmd/uplink/cmd/setup.go]: `n != 0` isn't needed here

Comment (mniewrzal) [reply cmd/uplink/cmd/setup.go]: ah, thanks

**Comment by mniewrzal** [comment]
whole PR it will be squashed before merging

File: cmd/uplink/cmd/setup.go [hunk 4]
```diff
@@ -71,19 +71,41 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
...
+				return err
+			}
 		}
 
 		// TODO add better validation
 		if satelliteAddress == "" {
-			return errs.New("Satellite address cannot be empty")
+			return errs.New("satellite address cannot be empty")
+		} else if len(satelliteAddress) == 1 {
+			switch satelliteAddress {
```
Comment (wthorp) [inline cmd/uplink/cmd/setup.go]: nit:  this logic could would be easier to maintain if it were indexed based (if we ever add other satellites)

Comment (mniewrzal) [reply cmd/uplink/cmd/setup.go]: yeah, current code is quite naive :) but I think we don't need anything special for now

**Pull request merged**